File: order-service/src/main/java/com/redthread/order/dto/AdminOrderDetailRes.java
package com.redthread.order.dto;

import java.math.BigDecimal;
import java.util.List;

public record AdminOrderDetailRes(
    Long id,
    String status,
    String userId,
    String fullAddress,
    BigDecimal totalAmount,
    List<AdminOrderItemRes> items
) {}

File: RedThread/app/src/main/java/com/example/redthread/data/remote/dto/AdminOrderDetailRes.kt
package com.example.redthread.data.remote.dto

import java.math.BigDecimal

data class AdminOrderDetailRes(
    val id: Long,
    val status: String,
    val userEmail: String,
    val fullAddress: String,
    val totalAmount: BigDecimal,
    val items: List<AdminOrderItemRes>
)
